viite-UI/src/view/navigationpanel/ProjectListModel.js

```diff
@@ -183,30 +194,31 @@
                     '<td style="width: 100px;" title="' + info + '">' + staticFieldProjectList(proj.statusDescription) + '</td>';
             switch (proj.statusCode) {
               case projectStatus.ErrorInViite.value:
-                html += '<td>' + '<button class="project-open btn btn-new-error" style="alignment: right; margin-bottom: 6px; margin-left: 25px; visibility: hidden">Avaa uudelleen</button>' + '</td>' +
+                  html += '<td>' + '<button class="project-open btn btn-new-error" style="alignment: right; margin-bottom: 6px; margin-left: 25px; visibility: hidden" data-projectStatus=' + proj.statusCode + '">Avaa uudelleen</button>' + '</td>' +
                     '</tr>';
                 break;
               case projectStatus.ErroredInTR.value:
-                html += '<td id="innerOpenProjectButton">' + '<button class="project-open btn btn-new-error" style="alignment: right; margin-bottom: 6px; margin-left: 25px" id="reopen-project-' + proj.id + '" value="' + proj.id + '">Avaa uudelleen</button>' + '</td>' +
+                  html += '<td id="innerOpenProjectButton">' + '<button class="project-open btn btn-new-error" style="alignment: right; margin-bottom: 6px; margin-left: 25px" id="reopen-project-' + proj.id + '" value="' + proj.id + '" data-projectStatus=' + proj.statusCode + '">Avaa uudelleen</button>' + '</td>' +
                     '</tr>';
                 break;
               default:
-                html += '<td id="innerOpenProjectButton">' + '<button class="project-open btn btn-new" style="alignment: right; margin-bottom: 6px; margin-left: 50px" id="open-project-' + proj.id + '" value="' + proj.id + '">Avaa</button>' + '</td>' +
+                  html += '<td id="innerOpenProjectButton">' + '<button class="project-open btn btn-new" style="alignment: right; margin-bottom: 6px; margin-left: 50px" id="open-project-' + proj.id + '" value="' + proj.id + '" data-projectStatus=' + proj.statusCode + '">Avaa</button>' + '</td>' +
```

Comment:
Is there missing or extra quotation mark in all of these three html appends? I count odd number of them. Just before "Avaa" or "Avaa uudelleen" there seems to be an extra one.